.circleci/config.yml

```diff
@@ -90,9 +90,19 @@ jobs:
             source emsdk_env.sh
             python scripts/test.py
 
+  build-docker-image:
+    machine: true
+    steps:
+      - checkout
+      - run: cd docker && make version=${CIRCLE_TAG}-upstream
+      - run: |
+          docker login -u "$DOCKER_USER" -p "$DOCKER_PASS"
+          docker push emscripten/emsdk:${CIRCLE_TAG}-upstream
+
 workflows:
+  version: 2
   flake8:
-    jobs:
+    jobs:ignore: /.*/
```

Comment:
What does this do?

Reply:
Sorry, that was a pretty sloppy mistake on my behalf